.github/workflows/npm-reanimated-package-build.yml

```diff
@@ -29,6 +29,8 @@ jobs:
   build:
     if: github.repository == 'software-mansion/react-native-reanimated'
     runs-on: ubuntu-latest
+    env:
+      PACKAGE_NAME:
```

Comment:
Shouldn't we define package name here? Correct me if I'm wrong, but at this moment it is empty

Reply:
This is just a placeholder so the .yml linter doesn't yell at me for using probably non-existing env variable 🤣